File: java/src/SumOfNumbers.java
public class SumOfNumbers {

    public static int GetSum(int a, int b) {
        int sum = 0;

        if (b < a) {
            int t = a;
            a = b;
            b = t;
        }

        for (int i = a; i <= b; i++) {
            sum += i;
        }

        return sum;
    }

    public static void main(String[] args) {
        int a = Integer.parseInt(args[0]);
        int b = Integer.parseInt(args[1]);

        System.out.println(GetSum(a, b));
    }
}

File: javascript/sum-of-numbers.js
function getSum(a, b) {
    let sum = 0;

    if (b < a) {
        const t = a;
        a = b;
        b = t;
    }

    for (let i = a; i <= b; i++) {
        sum += i;
    }

    return sum;
}

console.log(getSum(10, -5));